### DISCUSSION TITLE
Project scaling

### DISCUSSION THREAD

**Discussion by mvolf** [apache/camel-karavan#1160] [Q&A] [answered]
Hi @mgubaidullin,

Is anything planned for the ability to scale the running projects (e.g. defining number of pods to run the project on, managing them, defining scaling strategy etc.)?

Thanks.

**Comment by mgubaidullin** [comment, marked as answer]
You can add `deployment.jkube.yaml` to your project with required fragments.
Example: https://github.com/apache/camel-karavan/blob/4.4.0/karavan-app/src/main/resources/snippets/deployment.jkube.yaml
